Source/WebCore/rendering/updating/RenderTreeUpdater.cpp

```diff
@@ -342,8 +342,10 @@ void RenderTreeUpdater::updateElementRenderer(Element& element, const Style::Ele
     bool hasDisplayContents = elementUpdate.style->display() == DisplayType::Contents;
     if (hasDisplayContents)
         element.storeDisplayContentsStyle(makeUnique<RenderStyle>(WTFMove(elementUpdateStyle)));
-    else
+    else {
+        element.clearDisplayContentsStyle();
         element.resetComputedStyle();
+    }
```

Comment:
Should we now be calling `resetComputedStyle()` regardless of `hasDisplayContents`? Since that's now used purely for storing `getComputedStyle` results, and we have a render object now.

Reply:
You are right, resetComputedStyle here can be removed now.